src/fs_io.cc

```diff
@@ -10,6 +10,8 @@
 
 #include "squid.h"
 #include "comm/Loops.h"
+#include "compat/socket.h"
```

Comment:
The API in this file does not seem to be used here. Can it be dropped?
```suggestion
```

Reply:
yes. removing